package.json

```diff
@@ -115,6 +115,9 @@
     "tracked-built-ins": "^1.0.2",
     "tracked-toolbox": "^1.2.3"
   },
+  "dependencies": {
+    "@lblod/ember-vo-mu-file-upload": "git://github.com/lblod/ember-vo-mu-file-upload.git#feature/configurable-model-name"
```

Comment:
do we need this branch? looks like we don't but perhaps I'm missing something

Reply:
@Asergey91